js/components/modules/Workshops.jsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react'
+import {fetchWorkshops} from '../../actions'
+import {Link} from 'react-router'
+import Loading from './Loading'
+import Header from './Header'
+
+module.exports = React.createClass( {
+
+	componentDidMount: function () {
+		this.props.dispatch( fetchWorkshops() )
+	},
+
+	render: function () {
+
+		if ( ! this.props.posts.workshops.length ) {
+			return (
+				<div className="loading-wrap workshops">
+					<div className="loading"><span className="fa fa-heart"></span> Gathering workshops...</div>
+				</div>
+			)
+		}
+
+		return (
```

Comment:
@missjwo - Are workshops not stored as CPTs? This seems to be the only static content in the theme.